Changelog.md

```diff
@@ -4,6 +4,10 @@
 
 - Completion for `open-api` option in Camel K modeline filtered with JSON and YAML files
 - Completion for resource-like options (`open-api`, `property-file` and `resource`) in Camel K modeline is hiding IDE configuration files (which are in `.vscode` folder)
+- Document symbols for Camel routes written with Java DSL. It implies these features:

```

Comment:
I am not sure "implies" is a good choice of words here. Really we're not implying anything, we're providing a baseline for document symbol/outline support in the IDE. I think we can get rid of that last sentence entirely.